ci-operator/step-registry/baremetalds/devscripts/proxy/baremetalds-devscripts-proxy-commands.sh

```diff
@@ -40,9 +40,9 @@ EOF
 cat <<EOF> "${SHARED_DIR}/proxy-conf.sh"
 export HTTP_PROXY=http://${IP}:8213/
 export HTTPS_PROXY=http://${IP}:8213/
-export NO_PROXY="redhat.io,quay.io,redhat.com,openshift.com,svc,github.com,githubusercontent.com,google.com,googleapis.com,fedoraproject.org,localhost,127.0.0.1"
+export NO_PROXY="redhat.io,quay.io,redhat.com,openshift.com,svc,github.com,githubusercontent.com,google.com,googleapis.com,fedoraproject.org,docker.io,localhost,127.0.0.1"
```

Comment:
what are we pulling from docker.io?  I seem to recall we got rid of docker.io registry usage in CI because of pull rate limits or something like that.

Is it just the FRR image?  If so I think we can drop this as I'm suggesting we drop the BGP job for now.

Reply:
This is for FRR image of BGP testing, so let's remove it.